RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.9.2
```

Comment:
Since it adds a new config key, it should be a minor version bump: 3.10.0

Reply:
Ech, i always forget about this...